IIASA_TLDR.md

```diff
@@ -0,0 +1,22 @@
+# docker-geoserver
+
+## IIASA specific TLDR
+
+ - copy `.env.tpl` to `.env` and fill in the proper admin password (`GEOSERVER_ADMIN_PASSWORD`)
+
+ - Extensions will be downloaded when present in `build_data/stable_plugins.txt` during the build step. A full list of available can be found in `build_data/stable_plugins_available.txt`
+
+ - Extensions can be enabled by editing `.env` (`STABLE_EXTENSIONS=plugin1,plugin2`)
+   Downloaded extensions will be installed when creating the container (no need to rebuild the image, just `docker-compose up`).
+
+ - The data directory location inside the container can be adjusted with `GEOSERVER_DATA_DIR`.
+   It will always be exposed on the host system under `./geoserver_data` (for data imports).
+
+ - Tomcat will be routed to Port `:8600` by docker.
+
```

Comment:
port needs to changed for prod (ene-geoserver-hotspots.iiasa.ac.at) 
and test (ece-geoserver-hotspots-test.iiasa.ac.at) geoservers running on hpg1002

Reply:
Good point!
The host system port is now configurable using the `.env` file.